push_notification/apps/subscriber/tests/unit/test_subscriber.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework.test import APITestCase
+from ....subscriber.services.subscriber_service import SubscriberService
+
+
+class TestSubscriberAPI(APITestCase):
+    def setUp(self):
+        self.payload = {
+            "endpoint": "https://fcm.googleapis.com/fcm/send/c1Nsz7N4gO8:APA91bExyTlmumh-",
+            "public_key": "BBckLc4pAr2pEwjx6Ho3tigyayn2XpuRc9JFHkQ-CghTevxk8HYmZAALlmChkfuHq6og9JlfDBPUAHBQ0OGwBiA",
+            "auth_key": "RbNu_79slmjxSMZOmyYzgQ"
+        }
+        self.url = "/api/v1/subscriber/"
```

Comment:
You can use `reverse` method of Django instead of hardcoding URL. This is possible as you are using `name` field in `urls.py`